dev/devicelab/bin/run.dart

```diff
@@ -111,7 +118,9 @@ Future<void> _runTasks() async {
 
     if (serviceAccountTokenFile != null) {
       final Cocoon cocoon = Cocoon(serviceAccountTokenPath: serviceAccountTokenFile);
-      await cocoon.sendTaskResult(taskName: taskName, result: result);
+      /// Cocoon's definition of a task is more specific than [taskName], and to
+      /// upload we instead send the [luciBuilder] this is running on.
+      await cocoon.sendTaskResult(taskName: luciBuilder, result: result);
```

Comment:
`taskName: luciBuilder`

Somewhere there's going to be this friction in how we connect a DeviceLab test runner taskName and a Cocoon datastore taskName. I chose here to minimize confusion as this is one of the few areas in the code where Cocoon and DeviceLab test runner are connected.